README.md

```diff
@@ -1,27 +1,26 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+LeaderBoard Project
 
+> LeaderBoard website is a website that allows a user to input his name and the user score and it is saved and displayed and APIs will be used as well
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- html
+- css
+- webpack
+- npm
+- es6 modules & syntax
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
 
-
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
```

Comment:
- [ ] Kindly make sure your README is professional and contains all the information about how to run the app with the API(s) it consumes. Please provide a clear instruction about how to use the app(npm i, npm start, etc...).

Reply:
readMe file modified and corrected